wdl-lint/RULES.md

```diff
@@ -41,6 +41,7 @@ be out of sync with released packages.
 | `PreambleCommentAfterVersion`    | Clarity                       | Ensures that documents have correct comments in the preamble.                                     |
 | `PreambleFormatting`             | Spacing, Style, Clarity       | Ensures that documents have correct whitespace in the preamble.                                   |
 | `RuntimeSectionKeys`             | Completeness, Deprecated      | Ensures that runtime sections have the appropriate keys.                                          |
+| `RedundantInputAssignment`       | Clarity, Correctness          | Ensures that redundant input assignments are shortened                                            |
```

Comment:
@a-frantz just made my best guess, but hoping these tags are correct

Reply:
I don't think `Correctness` applies. I do think `Style` should be added. I'm a little on the fence about `Clarity`. Leaning towards removing that as well, and leaving it `Style` by itself. If anyone else reviews this, I'll let them weigh in.